api/app/signals/apps/signals/migrations/0126_update_history_after_directing_dept_remove.py

```diff
@@ -0,0 +1,139 @@
+from django.db import migrations
+
```

Comment:
This needs to be fixed. A couple of changes has been made to the history and we will loose them when merging this PR.

Reply:
Since we are using 'old' style history, used table rename instead of re-create